LiskHQ/lisk-mobile
# Implement the new design of initialisation modal - Closes #855

# What was the bug or feature?
Described in #855.

### How did I fix it?
Created `InitializationModal` component and added the logic for it on `Home`.

NOTE: `account.initialized` is not available immediately after signing in so the modal is called through `setTimeOut`.

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
Create a new user, transfer at least 0.2 LSK and sign in with it. The modal should appear on the home screen.


# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
